django/core/files/move.py

```diff
@@ -66,7 +66,13 @@ def file_move_safe(old_file_name, new_file_name, chunk_size=1024 * 64, allow_ove
         finally:
             locks.unlock(fd)
             os.close(fd)
-    copystat(old_file_name, new_file_name)
+
+    try:
+        copystat(old_file_name, new_file_name)
+    except PermissionError as e:
```

Comment:
@djackson-saa and I investigated the `PermissionError` and it was only added to Python in version 3.3 as a subclass of `OSError` and **not** available in Python2.7. In order to make this Python2.7 compatible, we are going to change the class we are excepting to `OSError` to cover all bases.